cmd/ui/src/App.tsx

```diff
@@ -86,19 +88,39 @@ export const Inner: React.FC = () => {
                             outline: `1px solid ${theme.palette.color.links}`,
                         },
                     },
+                    '& .models': {
```

Comment:
Above '& .models', would you please add this styling for updating the "required" red font color 🙏 
```
                    '& .parameter__name.required::after': {
                        color: theme.palette.color.error,
                    },
```

Reply:
good catch thanks man. updated.